config/prow/config.yaml

```diff
@@ -74,3 +74,53 @@ sinker:
   max_prowjob_age: 2h
   max_pod_age: 2h
   terminated_pod_ttl: 30m
+
+presets:
```

Comment:
this whole section should not be here but rather moved to the job definition itself where used. i.e https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presubmits-main.yaml#L1-L83

Reply:
No it is in the right place. Check https://github.com/kubernetes/test-infra/blob/master/config/prow/config.yaml#L980